prepare_dataset.py

```diff
@@ -338,7 +338,7 @@ def prepare_dataset(gcp_bucket, config_file, random_module_global_seed, numpy_ra
         all_scans += scans
     all_scans = sorted(set(all_scans))
 
-    assert not remote_folder_exists(prepared_dataset_remote_dest, dataset_id, sample_file_name='config.yaml')
+    assert not remote_folder_exists(prepared_dataset_remote_dest, dataset_id, sample_file_name='config.yaml'), "dataset already exists in the gcp_bucket; change name of dataset in config directory"
 
```

Comment:
Let's revise message to:
```
"Dataset already exists in the GCP bucket. Select new name in dataset config used here. Do not modify or delete the existing dataset in the GCP bucket."
```

Reply:
Think I changed the message, but I'm not seeing it pop up on the git web browser after committing via git desktop. I'll re-request the review, so let me know if you can see the change populate on here!